cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/plugin/PluginInstantiator.java

```diff
@@ -117,7 +117,6 @@ public PluginInstantiator(CConfiguration cConf, ClassLoader parentClassLoader, F
     this.instantiatorFactory = new InstantiatorFactory(false);
     File tmpDir = new File(cConf.get(Constants.CFG_LOCAL_DATA_DIR),
                            cConf.get(Constants.AppFabric.TEMP_DIR)).getAbsoluteFile();
-
```

Comment:
nit: looks like accidental change. This is the only change in this class

Reply:
Will remove

Reply:
Removed